lighthouse-cli/run.js

```diff
@@ -218,10 +218,23 @@ async function runLighthouse(url, flags, config) {
     await potentiallyKillChrome(launchedChrome);
     process.removeListener('unhandledRejection', handleTheUnhandled);
 
+    // Runtime errors indicate something was *very* wrong with the page result.
+    // We don't want the user to have to parse the report to figure it out, so we'll still exit
+    // with an error code after we saved the results.
+    if (runnerResult && runnerResult.lhr.runtimeError) {
+      const {runtimeError} = runnerResult.lhr;
+      return printErrorAndExit({
+        name: 'LHError',
+        friendlyMessage: runtimeError.message,
```

Comment:
do we want to include that we've saved the results, as well? I guess we have the normal save log entries, but it could be helpful?

Reply:
hm, where are you thinking of adding it? 

I'm ideally thinking it might be like `blah blah usual error message, and your assets have still be saved to disk` or something, but mucking with the error message seems like more trouble than its worth